Lidgren.Network/NetPeer.Logging.cs

```diff
@@ -24,7 +24,7 @@ namespace Lidgren.Network
 {
 	public partial class NetPeer
 	{
-		internal event Action<NetIncomingMessageType, string> LogEvent; 
+		internal event Action<NetIncomingMessageType, string> LogEvent = delegate { };

```

Comment:
```suggestion
		internal event Action<NetIncomingMessageType, string>? LogEvent;
```

PS. I've never seen that delegate syntax before, and I'm scared.

Reply:
That is a quite common way to eliminate a default null value on a non-nullable event, by assigning an empty delegate.
But since you're so scared, I went ahead and tested the `+=` operator on a nullable event field and it somehow works just fine.
It breaks the principle of least surprise, for fyi.

Reply:
Making the event nullable is the correct thing to do in the first place. It's just a quirk of how events are stored internally.